src/module-3/online-store/scss/base/_layout.scss

```diff
@@ -0,0 +1,44 @@
+.overlap-main {
+  background-color: #f9f9f9;
+  box-shadow: 0 2px 8px #00000014;
+  display: flex;
+  flex-direction: column;
+  padding: 27px 33px;
+  width: fit-content;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.flex-row-main {
+  display: flex;
+  margin-top: 43px;
+}
+
+.flex-col-right {
+  display: flex;
+  flex-direction: column;
+  min-width: 25%;
+}
+
+.flex-col-grid {
+  display: flex;
+  flex-direction: column;
+  margin-left: 27px;
+}
+
+.flex-row {
+  display: flex;
```

Comment:
you may add code you use frequently to mixins, for example, there can be:  
`@mixin set-flex($direction: row, $justify: flex-start, $align: stretch) { `  
  `dislpay: flex;`  
  `flex-direction: $direction;`  
  `justify-content: $justifyt;`
  `align-items: $align;`  
`}`